Unable to perform integration check

**Describe the bug**
I just installed via the config available here: https://github.com/elninotech/uppload/blob/master/.github/workflows/merge-pr.yml

None of the merge checks are working, with this error:

```
##[error]Cannot read property 'head' of undefined
Run squalrus/merge-bot@v0.2.0
  with:
    GITHUB_TOKEN: ***
    test: true
    reviewers: true
    blocking-labels: do not merge
    checks: true
    method: squash
    labels: ready
test mode: true
required reviewers: true
required labels: ["ready"]
blocking labels: ["do not merge"]
merge method: squash
##[error]Cannot read property 'head' of undefined

```

Here are the logs: https://github.com/elninotech/uppload/commit/20acfb19cd21f074dee8d0618ba2688c6da29c41/checks?check_suite_id=480268098

The logs are here from that merge: https://github.com/elninotech/uppload/runs/472090462?check_suite_focus=true

```json
payload: {
  "avatar_url": "https://avatars2.githubusercontent.com/oa/1508?v=4",
  "branches": [
    {
      "commit": {
        "sha": "a97a890bd526ba0b2cb0c1a3f4f3d7864d64d054",
        "url": "https://api.github.com/repos/elninotech/uppload/commits/a97a890bd526ba0b2cb0c1a3f4f3d7864d64d054"
      },
      "name": "master",
      "protected": false
    }
  ],
  "commit": {
    "author": {
      "avatar_url": "https://avatars3.githubusercontent.com/u/2841780?v=4",
      "events_url": "https://api.github.com/users/AnandChowdhary/events{/privacy}",
      "followers_url": "https://api.github.com/users/AnandChowdhary/followers",
      "following_url": "https://api.github.com/users/AnandChowdhary/following{/other_user}",
      "gists_url": "https://api.github.com/users/AnandChowdhary/gists{/gist_id}",
      "gravatar_id": "",
      "html_url": "https://github.com/AnandChowdhary",
      "id": 2841780,
      "login": "AnandChowdhary",
      "node_id": "MDQ6VXNlcjI4NDE3ODA=",
      "organizations_url": "https://api.github.com/users/AnandChowdhary/orgs",
      "received_events_url": "https://api.github.com/users/AnandChowdhary/received_events",
      "repos_url": "https://api.github.com/users/AnandChowdhary/repos",
      "site_admin": false,
      "starred_url": "https://api.github.com/users/AnandChowdhary/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/AnandChowdhary/subscriptions",
      "type": "User",
      "url": "https://api.github.com/users/AnandChowdhary"
    },
    "comments_url": "https://api.github.com/repos/elninotech/uppload/commits/a97a890bd526ba0b2cb0c1a3f4f3d7864d64d054/comments",
    "commit": {
      "author": {
        "date": "2020-02-27T10:17:48Z",
        "email": "anandchowdhary@gmail.com",
        "name": "Anand Chowdhary"
      },
      "comment_count": 0,
      "committer": {
        "date": "2020-02-27T10:17:48Z",
        "email": "noreply@github.com",
        "name": "GitHub"
      },
      "message": "Merge pull request #95 from squalrus/master\n\nUpdate merge-bot to v0.4.1",
      "tree": {
        "sha": "12618237f38c2651b13421cead1fd5597587ba4d",
        "url": "https://api.github.com/repos/elninotech/uppload/git/trees/12618237f38c2651b13421cead1fd5597587ba4d"
      },
      "url": "https://api.github.com/repos/elninotech/uppload/git/commits/a97a890bd526ba0b2cb0c1a3f4f3d7864d64d054",
      "verification": {
        "payload": "tree 12618237f38c2651b13421cead1fd5597587ba4d\nparent 86f7659f6a425dca376108cd85e9d22809157b41\nparent d5d6def5ea57f46ed4be53cdb535410cf038b05b\nauthor Anand Chowdhary <anandchowdhary@gmail.com> 1582798668 +0100\ncommitter GitHub <noreply@github.com> 1582798668 +0100\n\nMerge pull request #95 from squalrus/master\n\nUpdate merge-bot to v0.4.1",
        "reason": "valid",
        "signature": "-----BEGIN PGP SIGNATURE-----\n\nwsBcBAABCAAQBQJeV5dMCRBK7hj4Ov3rIwAAdHIIALA3K1kyut1EBBRXfpxYLuLZ\nIPuqWB3/K4hgjSGnuYNueLiEl5RZvkFLk45zapu9LBYOI+5rWy9KyQCrXrbtHU6x\nzWTi1dVZoiYYQYTPxS6ynhRKLNOonLFaVsJG/KN0xE5/O3Gxw7fi5r1hGAocro0L\n2hs3Mz8J1WOzXVGK2G9Gq1EeRLcvw/KgGyqgpWwkt8RCLP1djddJzOsSWWn6nWCx\n76wACqBw/TxrOwAFaEazniuAYNoi8muwQD8EGY4q9/7CuDDKgKYxP9ZfhBmOQJ0q\njZLKEOguz7/oPnjefMGF2ptDYml1XVLu8wKZaKR5U/amaNnXRI8eY0M/Z7AINZE=\n=QnWb\n-----END PGP SIGNATURE-----\n",
        "verified": true
      }
    },
    "committer": {
      "avatar_url": "https://avatars3.githubusercontent.com/u/19864447?v=4",
      "events_url": "https://api.github.com/users/web-flow/events{/privacy}",
      "followers_url": "https://api.github.com/users/web-flow/followers",
      "following_url": "https://api.github.com/users/web-flow/following{/other_user}",
      "gists_url": "https://api.github.com/users/web-flow/gists{/gist_id}",
      "gravatar_id": "",
      "html_url": "https://github.com/web-flow",
      "id": 19864447,
      "login": "web-flow",
      "node_id": "MDQ6VXNlcjE5ODY0NDQ3",
      "organizations_url": "https://api.github.com/users/web-flow/orgs",
      "received_events_url": "https://api.github.com/users/web-flow/received_events",
      "repos_url": "https://api.github.com/users/web-flow/repos",
      "site_admin": false,
      "starred_url": "https://api.github.com/users/web-flow/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/web-flow/subscriptions",
      "type": "User",
      "url": "https://api.github.com/users/web-flow"
    },
    "html_url": "https://github.com/elninotech/uppload/commit/a97a890bd526ba0b2cb0c1a3f4f3d7864d64d054",
    "node_id": "MDY6Q29tbWl0MTMwMjI3NDQyOmE5N2E4OTBiZDUyNmJhMGIyY2IwYzFhM2Y0ZjNkNzg2NGQ2NGQwNTQ=",
    "parents": [
      {
        "html_url": "https://github.com/elninotech/uppload/commit/86f7659f6a425dca376108cd85e9d22809157b41",
        "sha": "86f7659f6a425dca376108cd85e9d22809157b41",
        "url": "https://api.github.com/repos/elninotech/uppload/commits/86f7659f6a425dca376108cd85e9d22809157b41"
      },
      {
        "html_url": "https://github.com/elninotech/uppload/commit/d5d6def5ea57f46ed4be53cdb535410cf038b05b",
        "sha": "d5d6def5ea57f46ed4be53cdb535410cf038b05b",
        "url": "https://api.github.com/repos/elninotech/uppload/commits/d5d6def5ea57f46ed4be53cdb535410cf038b05b"
      }
    ],
    "sha": "a97a890bd526ba0b2cb0c1a3f4f3d7864d64d054",
    "url": "https://api.github.com/repos/elninotech/uppload/commits/a97a890bd526ba0b2cb0c1a3f4f3d7864d64d054"
  },
  "context": "continuous-integration/travis-ci/push",
  "created_at": "2020-02-27T10:19:43+00:00",
  "description": "The Travis CI build passed",
  "id": 8941919094,
  "name": "elninotech/uppload",
  "organization": {
    "avatar_url": "https://avatars3.githubusercontent.com/u/2952546?v=4",
    "description": "",
    "events_url": "https://api.github.com/orgs/elninotech/events",
    "hooks_url": "https://api.github.com/orgs/elninotech/hooks",
    "id": 2952546,
    "issues_url": "https://api.github.com/orgs/elninotech/issues",
    "login": "elninotech",
    "members_url": "https://api.github.com/orgs/elninotech/members{/member}",
    "node_id": "MDEyOk9yZ2FuaXphdGlvbjI5NTI1NDY=",
    "public_members_url": "https://api.github.com/orgs/elninotech/public_members{/member}",
    "repos_url": "https://api.github.com/orgs/elninotech/repos",
    "url": "https://api.github.com/orgs/elninotech"
  },
  "repository": {
    "archive_url": "https://api.github.com/repos/elninotech/uppload/{archive_format}{/ref}",
    "archived": false,
    "assignees_url": "https://api.github.com/repos/elninotech/uppload/assignees{/user}",
    "blobs_url": "https://api.github.com/repos/elninotech/uppload/git/blobs{/sha}",
    "branches_url": "https://api.github.com/repos/elninotech/uppload/branches{/branch}",
    "clone_url": "https://github.com/elninotech/uppload.git",
    "collaborators_url": "https://api.github.com/repos/elninotech/uppload/collaborators{/collaborator}",
    "comments_url": "https://api.github.com/repos/elninotech/uppload/comments{/number}",
    "commits_url": "https://api.github.com/repos/elninotech/uppload/commits{/sha}",
    "compare_url": "https://api.github.com/repos/elninotech/uppload/compare/{base}...{head}",
    "contents_url": "https://api.github.com/repos/elninotech/uppload/contents/{+path}",
    "contributors_url": "https://api.github.com/repos/elninotech/uppload/contributors",
    "created_at": "2018-04-19T14:23:20Z",
    "default_branch": "master",
    "deployments_url": "https://api.github.com/repos/elninotech/uppload/deployments",
    "description": "📁 Uppload is a better JavaScript image uploader with 30+ plugins",
    "disabled": false,
    "downloads_url": "https://api.github.com/repos/elninotech/uppload/downloads",
    "events_url": "https://api.github.com/repos/elninotech/uppload/events",
    "fork": false,
    "forks": 53,
    "forks_count": 53,
    "forks_url": "https://api.github.com/repos/elninotech/uppload/forks",
    "full_name": "elninotech/uppload",
    "git_commits_url": "https://api.github.com/repos/elninotech/uppload/git/commits{/sha}",
    "git_refs_url": "https://api.github.com/repos/elninotech/uppload/git/refs{/sha}",
    "git_tags_url": "https://api.github.com/repos/elninotech/uppload/git/tags{/sha}",
    "git_url": "git://github.com/elninotech/uppload.git",
    "has_downloads": true,
    "has_issues": true,
    "has_pages": false,
    "has_projects": false,
    "has_wiki": false,
    "homepage": "https://uppload.js.org",
    "hooks_url": "https://api.github.com/repos/elninotech/uppload/hooks",
    "html_url": "https://github.com/elninotech/uppload",
    "id": 130227442,
    "issue_comment_url": "https://api.github.com/repos/elninotech/uppload/issues/comments{/number}",
    "issue_events_url": "https://api.github.com/repos/elninotech/uppload/issues/events{/number}",
    "issues_url": "https://api.github.com/repos/elninotech/uppload/issues{/number}",
    "keys_url": "https://api.github.com/repos/elninotech/uppload/keys{/key_id}",
    "labels_url": "https://api.github.com/repos/elninotech/uppload/labels{/name}",
    "language": "TypeScript",
    "languages_url": "https://api.github.com/repos/elninotech/uppload/languages",
    "license": {
      "key": "mit",
      "name": "MIT License",
      "node_id": "MDc6TGljZW5zZTEz",
      "spdx_id": "MIT",
      "url": "https://api.github.com/licenses/mit"
    },
    "merges_url": "https://api.github.com/repos/elninotech/uppload/merges",
    "milestones_url": "https://api.github.com/repos/elninotech/uppload/milestones{/number}",
    "mirror_url": null,
    "name": "uppload",
    "node_id": "MDEwOlJlcG9zaXRvcnkxMzAyMjc0NDI=",
    "notifications_url": "https://api.github.com/repos/elninotech/uppload/notifications{?since,all,participating}",
    "open_issues": 6,
    "open_issues_count": 6,
    "owner": {
      "avatar_url": "https://avatars3.githubusercontent.com/u/2952546?v=4",
      "events_url": "https://api.github.com/users/elninotech/events{/privacy}",
      "followers_url": "https://api.github.com/users/elninotech/followers",
      "following_url": "https://api.github.com/users/elninotech/following{/other_user}",
      "gists_url": "https://api.github.com/users/elninotech/gists{/gist_id}",
      "gravatar_id": "",
      "html_url": "https://github.com/elninotech",
      "id": 2952546,
      "login": "elninotech",
      "node_id": "MDEyOk9yZ2FuaXphdGlvbjI5NTI1NDY=",
      "organizations_url": "https://api.github.com/users/elninotech/orgs",
      "received_events_url": "https://api.github.com/users/elninotech/received_events",
      "repos_url": "https://api.github.com/users/elninotech/repos",
      "site_admin": false,
      "starred_url": "https://api.github.com/users/elninotech/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/elninotech/subscriptions",
      "type": "Organization",
      "url": "https://api.github.com/users/elninotech"
    },
    "private": false,
    "pulls_url": "https://api.github.com/repos/elninotech/uppload/pulls{/number}",
    "pushed_at": "2020-02-27T10:17:50Z",
    "releases_url": "https://api.github.com/repos/elninotech/uppload/releases{/id}",
    "size": 23681,
    "ssh_url": "git@github.com:elninotech/uppload.git",
    "stargazers_count": 623,
    "stargazers_url": "https://api.github.com/repos/elninotech/uppload/stargazers",
    "statuses_url": "https://api.github.com/repos/elninotech/uppload/statuses/{sha}",
    "subscribers_url": "https://api.github.com/repos/elninotech/uppload/subscribers",
    "subscription_url": "https://api.github.com/repos/elninotech/uppload/subscription",
    "svn_url": "https://github.com/elninotech/uppload",
    "tags_url": "https://api.github.com/repos/elninotech/uppload/tags",
    "teams_url": "https://api.github.com/repos/elninotech/uppload/teams",
    "trees_url": "https://api.github.com/repos/elninotech/uppload/git/trees{/sha}",
    "updated_at": "2020-02-27T10:17:52Z",
    "url": "https://api.github.com/repos/elninotech/uppload",
    "watchers": 623,
    "watchers_count": 623
  },
  "sender": {
    "avatar_url": "https://avatars3.githubusercontent.com/u/2841780?v=4",
    "events_url": "https://api.github.com/users/AnandChowdhary/events{/privacy}",
    "followers_url": "https://api.github.com/users/AnandChowdhary/followers",
    "following_url": "https://api.github.com/users/AnandChowdhary/following{/other_user}",
    "gists_url": "https://api.github.com/users/AnandChowdhary/gists{/gist_id}",
    "gravatar_id": "",
    "html_url": "https://github.com/AnandChowdhary",
    "id": 2841780,
    "login": "AnandChowdhary",
    "node_id": "MDQ6VXNlcjI4NDE3ODA=",
    "organizations_url": "https://api.github.com/users/AnandChowdhary/orgs",
    "received_events_url": "https://api.github.com/users/AnandChowdhary/received_events",
    "repos_url": "https://api.github.com/users/AnandChowdhary/repos",
    "site_admin": false,
    "starred_url": "https://api.github.com/users/AnandChowdhary/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/AnandChowdhary/subscriptions",
    "type": "User",
    "url": "https://api.github.com/users/AnandChowdhary"
  },
  "sha": "a97a890bd526ba0b2cb0c1a3f4f3d7864d64d054",
  "state": "success",
  "target_url": "https://travis-ci.org/elninotech/uppload/builds/655737447?utm_source=github_status&utm_medium=notification",
  "updated_at": "2020-02-27T10:19:43+00:00"
}
```

```json
config: {
  "blocking_labels": [
    "do not merge"
  ],
  "labels": [
    "ready"
  ],
  "merge_method": "squash",
  "review_required": true,
  "checks_enabled": true,
  "test_mode": true
}
```

```
##[error]Cannot read property 'labels' of undefined
```

--- a/.github/workflows/merge-bot.yml
+++ b/.github/workflows/merge-bot.yml
@@ -26,6 +26,6 @@ jobs:
         test: false
         reviewers: false
         labels: ready
-        blocking-labels: do not merge
-        checks: false
+        blocking_labels: do not merge
+        checks_enabled: false
         method: squash

--- a/README.md
+++ b/README.md
@@ -28,7 +28,7 @@ One or more labels required for integration. Default is `"ready"`.
 
 ![merge and sign-off GitHub labels](./assets/labels.png)
 
-### `blocking-labels`
+### `blocking_labels`
 
 One or more labels that block the integration. Default is `"do not merge"`.
 
@@ -75,7 +75,7 @@ jobs:
         test: true
         reviewers: true
         labels: ready, merge
-        blocking-labels: do not merge
-        checks: true
+        blocking_labels: do not merge
+        checks_enabled: true
         method: squash
 ```

--- a/__mocks__/config/config-checks.js
+++ b/__mocks__/config/config-checks.js
@@ -1,0 +1,8 @@
+module.exports = {
+    "blocking_labels": ["do not merge"],
+    "labels": ["ready", "merge"],
+    "method": "merge",
+    "reviewers": true,
+    "checks_enabled": true,
+    "test": false
+};

--- a/__mocks__/config/config-default.js
+++ b/__mocks__/config/config-default.js
@@ -1,8 +1,8 @@
 module.exports = {
     "blocking_labels": ["do not merge"],
     "labels": ["ready", "merge"],
-    "merge_method": "squash",
+    "merge_method": "merge",
     "review_required": true,
-    "checks_enabled": true,
-    "test_mode": true
+    "checks_enabled": false,
+    "test_mode": false
 };

--- a/__mocks__/config/core-complex.js
+++ b/__mocks__/config/core-complex.js
@@ -1,7 +1,7 @@
 class CoreComplex {
     getInput(key) {
         switch (key) {
-            case 'blocking-labels':
+            case 'blocking_labels':
                 return 'hold,  stop,no';
 
             case 'labels':

--- a/__mocks__/config/core-default.js
+++ b/__mocks__/config/core-default.js
@@ -1,23 +1,23 @@
 class CoreDefault {
     getInput(key) {
         switch (key) {
-            case 'blocking-labels':
+            case 'blocking_labels':
                 return 'do not merge';
 
             case 'labels':
                 return 'ready, merge';
 
             case 'method':
-                return 'squash';
+                return 'merge';
 
             case 'reviewers':
                 return 'true';
 
             case 'checks_enabled':
-                return 'true';
+                return 'false';
 
             case 'test':
-                return 'true';
+                return 'false';
 
             default:
                 break;

--- a/__mocks__/message/message-expected.js
+++ b/__mocks__/message/message-expected.js
@@ -13,7 +13,7 @@ module.exports = `### merge bot test mode
 <tr><td>required label(s)</td><td>[\"ready\",\"merge\"]</td></tr>
 <tr><td>blocking label(s)</td><td>[\"do not merge\"]</td></tr>
 <tr><td>reviewers required</td><td>true</td></tr>
-<tr><td>merge method</td><td>squash</td></tr>
+<tr><td>merge method</td><td>merge</td></tr>
 </table>
 
 #### pull request details

--- a/__tests__/checks.test.js
+++ b/__tests__/checks.test.js
@@ -1,7 +1,7 @@
 const Pull = require('../lib/pull');
 
 const payloadDefault = require('../__mocks__/pull/payload-default');
-const configDefault = require('../__mocks__/config/config-default');
+const configChecks = require('../__mocks__/config/config-checks');
 
 const checks0 = require('../__mocks__/checks/check-0');
 const checks1a = require('../__mocks__/checks/check-1a');
@@ -17,68 +17,68 @@ test('empty checks data', () => {
     const pull = new Pull(payloadDefault);
     pull.compileChecks(checks0);
 
-    expect(pull.isChecksComplete(configDefault.checks_enabled)).toBe(false);
+    expect(pull.isChecksComplete(configChecks.checks_enabled)).toBe(false);
 });
 
 test('checks data queued, in_progress, completed (success)', () => {
     const pull = new Pull(payloadDefault);
     pull.compileChecks(checks1a);
 
-    expect(pull.isChecksComplete(configDefault.checks_enabled)).toBe(false);
+    expect(pull.isChecksComplete(configChecks.checks_enabled)).toBe(false);
 });
 
 test('checks data queued, in_progress, completed (failure)', () => {
     const pull = new Pull(payloadDefault);
     pull.compileChecks(checks1b);
 
-    expect(pull.isChecksComplete(configDefault.checks_enabled)).toBe(false);
+    expect(pull.isChecksComplete(configChecks.checks_enabled)).toBe(false);
 });
 
 test('checks data in_progress, completed 2x (success)', () => {
     const pull = new Pull(payloadDefault);
     pull.compileChecks(checks2a);
 
-    expect(pull.isChecksComplete(configDefault.checks_enabled)).toBe(true);
+    expect(pull.isChecksComplete(configChecks.checks_enabled)).toBe(true);
 });
 
 test('checks data in_progress, completed 2x (success, failure)', () => {
     const pull = new Pull(payloadDefault);
     pull.compileChecks(checks2b);
 
-    expect(pull.isChecksComplete(configDefault.checks_enabled)).toBe(false);
+    expect(pull.isChecksComplete(configChecks.checks_enabled)).toBe(false);
 });
 
 test('checks data in_progress, completed 2x (failure)', () => {
     const pull = new Pull(payloadDefault);
     pull.compileChecks(checks2c);
 
-    expect(pull.isChecksComplete(configDefault.checks_enabled)).toBe(false);
+    expect(pull.isChecksComplete(configChecks.checks_enabled)).toBe(false);
 });
 
 test('checks data completed 3x (success)', () => {
     const pull = new Pull(payloadDefault);
     pull.compileChecks(checks3a);
 
-    expect(pull.isChecksComplete(configDefault.checks_enabled)).toBe(true);
+    expect(pull.isChecksComplete(configChecks.checks_enabled)).toBe(true);
 });
 
 test('checks data completed 3x (failure)', () => {
     const pull = new Pull(payloadDefault);
     pull.compileChecks(checks3b);
 
-    expect(pull.isChecksComplete(configDefault.checks_enabled)).toBe(false);
+    expect(pull.isChecksComplete(configChecks.checks_enabled)).toBe(false);
 });
 
 test('checks data in_progress, completed 6x (success)', () => {
     const pull = new Pull(payloadDefault);
     pull.compileChecks(checks4a);
 
-    expect(pull.isChecksComplete(configDefault.checks_enabled)).toBe(true);
+    expect(pull.isChecksComplete(configChecks.checks_enabled)).toBe(true);
 });
 
 test('checks data in_progress, completed 6x (success) + checks_enabled = false', () => {
     const pull = new Pull(payloadDefault);
     pull.compileChecks(checks4a);
 
-    expect(pull.isChecksComplete(false)).toBe(false);
+    expect(pull.isChecksComplete(false)).toBe(true);
 });

--- a/action.yml
+++ b/action.yml
@@ -12,7 +12,7 @@ inputs:
     description: 'Labels required for integration'
     required: true
     default: 'ready'
-  blocking-labels:
+  blocking_labels:
     description: 'Labels that block the integration'
     required: false
     default: 'do not merge'

--- a/lib/config.js
+++ b/lib/config.js
@@ -1,6 +1,6 @@
 class Config {
     constructor(core) {
-        this.blocking_labels = core.getInput('blocking-labels').split(',').map(x => x.trim());
+        this.blocking_labels = core.getInput('blocking_labels').split(',').map(x => x.trim());
         this.labels = core.getInput('labels').split(',').map(x => x.trim());
         this.merge_method = core.getInput('method');
         this.review_required = core.getInput('reviewers') === 'true';

--- a/lib/pull.js
+++ b/lib/pull.js
@@ -74,7 +74,11 @@ class Pull {
      * @param {boolean} checks_enabled is the checks_enabled config enabled
      */
     isChecksComplete(checks_enabled) {
-        if (this.checks.total === 0 || !checks_enabled){
+        if (!checks_enabled) {
+            return true;
+        }
+
+        if (this.checks.total === 0){
             return false;
         }
 

